cmake/secp256k1/src/ecmult_static_context.h

```diff
@@ -0,0 +1,1160 @@
+#ifndef _SECP256K1_ECMULT_STATIC_CONTEXT_
```

Comment:
It is generated by a tool included in secp256k1. Instead of computing the context at runtime, it is computed at build time and include as static data.